docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -245,16 +255,71 @@ slow to return results.
 (such as AWS Athena and BigQuery). Repeatedly querying for this data can easily
 rack up costs.
 
+## Optimizing Pre-Aggregation Build Times
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| For ideal performance, pre-aggregations should be built using a dedicated
+| Refresh Worker. [See here for more details][ref-prod-list-refresh].
+<!-- prettier-ignore-end -->
+
+By default, Cube.js will use the source database as a temporary staging area for
+writing pre-aggregations before moving them to Cube Store.
+
+> Get more info about default strategy
```

Comment:
When streaming is not supported in a driver and it's not configured for export, cube.js will download the whole data from a table into memory, after that it will insert data.

How we insert data depends on a driver.

- In the MySQL driver, we are using batch inserts (1k rows)
- In the Postgres driver, we insert all in one query.
- In the Cube Store driver, we are using batch inserts (2k rows)